File: CSHARP/Cap07/c07ex06.cs
// c07ex06.cs

using System;
using static System.Console;

class Program
{
    static int Fatorial(int N)
    {
        int Fat = 1;
        for (int I = 1; I <= N; I++)
            Fat *= I;
        return Fat;
    }

    public static void Main(string[] args)
    {
        int Limite;

        Write("Qual fatorial: ");
        Limite = int.Parse(ReadLine());

        WriteLine($"Fatorial = {Fatorial(Limite)}");

        Write("\nTecle <Enter> para encerrar... ");
        ReadLine();
    }
}

File: KOTLIN/Cap07/c07ex06.kt
// c07ex06.kt

fun fatorial(N: Int): Int {
    var fat = 1
    for (i in 1..N) {
        fat *= i
    }
    return fat
}

fun main() {
    var limite: Int

    print("Qual fatorial: ")
    limite = readLine()?.toIntOrNull() ?: 0

    println("Fatorial = ${fatorial(limite)}")

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
